packages/react/src/components/AspectRatio/AspectRatio.js

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { settings } from 'carbon-components';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import React from 'react';
+import useMatchMedia from '../../internal/useMatchMedia';
+
+const { prefix } = settings;
+const ratios = ['16x9', '2x1', '4x3', '1x1', '1x2'];
```

Comment:
Is it possible to add `2x1` as an aspect ratio, we're using this on the IDL homepage and have it added to the Gatsby theme.